java/src/org/openqa/selenium/grid/sessionqueue/SessionNotCreated.java

```diff
@@ -54,7 +56,7 @@ public HttpResponse execute(HttpRequest req) throws UncheckedIOException {
       SessionNotCreatedException exception = new SessionNotCreatedException(message);
       queue.complete(requestId, Either.left(exception));
 
-      HttpResponse res = new HttpResponse();
+      HttpResponse res = new HttpResponse().setContent(asJson(singletonMap("value", false)));
```

Comment:
It might be better to pass the return value of `queue.complete` to the response to avoid having a hard coded value.

Reply:
I put a hard coded value because the value did not matter
But, You're right, better to pass the method reply